smac/runhistory/runhistory.py

```diff
@@ -17,14 +17,24 @@
 __version__ = "0.0.1"
 
 
-RunKey = collections.namedtuple(
-    'RunKey', ['config_id', 'instance_id', 'seed'])
+# RunKey = collections.namedtuple(
+#     'RunKey', ['config_id', 'instance_id', 'seed', 'budget'])
+# NOTE class instead of collection to have a default value for budget in RunKey
+class RunKey(collections.namedtuple('RunKey', ['config_id', 'instance_id', 'seed', 'budget'])):
+    __slots__ = ()
+
+    def __new__(cls, config_id, instance_id, seed, budget=0.0):
+        return super().__new__(cls, config_id, instance_id, seed, budget)
 
 
 InstSeedKey = collections.namedtuple(
```

Comment:
Is this one still necessary now that we also store the budget?

Reply:
It isn't. But I thought I will let it be in case we need it again in the future.

Reply:
Please don't keep code because one could need it in the future. If we need it again we can retrieve it from the git history.

Reply:
Apologies, I was wrong. We still use in another variable where we map configs, inst-seeds to budget.